examples/megatron-models/README.md

```diff
@@ -0,0 +1,115 @@
+<!---
+# ##############################################################################################
+# 
+# Copyright (c) 2021-, NVIDIA CORPORATION.  All rights reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# 
+# ##############################################################################################
+-->
+
+# How to run Megatron BERT and GPT2 using Transformers
```

Comment:
It sounds good with me. 

Where do you want me to put it? `docs/sources/model_doc`? Or, `model_cards/nvidia/megatron-models`?

Reply:
I was thinking of a modelcard on the hub directly! It seems you're already in the NVIDIA organization there: https://huggingface.co/nvidia

The idea would be to create a repository there containing only the model card with the contents of the README you've put here. Does that make sense? When looking for models, that's where our users look, so that's the easiest way to enable discoverability.

Reply:
I've created two model cards. One for [BERT](https://huggingface.co/nvidia/megatron-bert) and one for [GPT2](https://huggingface.co/nvidia/megatron-gpt2).